Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ui/main/home/HomeFragmentViewModel.kt

```diff
@@ -206,7 +208,7 @@ class HomeFragmentViewModel @AssistedInject constructor(
         mutableListOf<HomeItem>().apply {
             Timber.d("rampDownNotice=%s", rampDownNotice)
             if (rampDownNotice?.visible == true) {
-                // TODO addRampDownCard(rampDownNotice)
+                addEolCard(rampDownNotice)
```

Comment:
Please rename to `addRampDownCard` I know I did not like that at the beginning ,but to avoid confusion about EOL (End Of Life) I used the same term as in specs

Reply:
changed